README.md-edits

```diff
@@ -1,5 +1,5 @@
 e<header>
-LLLLLLLLLLLL
+KKKKKKKKKKKK
```

Comment:
lalalala

Reply:
[Moderators.docx](https://github.com/LefterisGO/skills-introduction-to-githhub/files/12660962/Moderators.docx)
 mod